lib/sync-DataSetModel.js

```diff
@@ -156,6 +159,18 @@ var self = {
     });
   },
 
+  doResponseInterceptor: function (dataset_id, params, cb) {
+    self.getDataset(dataset_id, function (err, dataset) {
+      if (err) return cb(err);
```

Comment:
Don't mind our little robotic friend, he's there as a sanity check to spot code style issues among other things! Feel free to adhere to it for clarity where it makes sense but it's not mandatory.